rules/windows/initial_access_unusual_dns_service_children.toml

```diff
@@ -23,12 +23,50 @@ license = "Elastic License v2"
 
-### Investigating Unusual Child Process
-Detection alerts from this rule indicate potential suspicious child processes spawned after exploitation from CVE-2020-1350 (SigRed) has occurred. Here are some possible avenues of investigation:
-- Any suspicious or abnormal child process spawned from dns.exe should be reviewed and investigated with care. It's impossible to predict what an adversary may deploy as the follow-on process after the exploit, but built-in discovery/enumeration utilities should be top of mind (whoami.exe, netstat.exe, systeminfo.exe, tasklist.exe).
-- Built-in Windows programs that contain capabilities used to download and execute additional payloads should also be considered. This is not an exhaustive list, but ideal candidates to start out would be: mshta.exe, powershell.exe, regsvr32.exe, rundll32.exe, wscript.exe, wmic.exe.
-- If the DoS exploit is successful and DNS Server service crashes, be mindful of potential child processes related to werfault.exe occurring.
-- Any subsequent activity following the child process spawned related to execution/network activity should be thoroughly reviewed from the endpoint.
+### Investigating Unusual Child Process of dns.exe
+
+SIGRed (CVE-2020-1350) is a wormable, critical vulnerability in the Windows DNS server that affects Windows Server
+versions 2003 to 2019, and can be triggered by a malicious DNS response. As the service is running in elevated privileges
+(SYSTEM), if exploited successfully, an attacker is granted Domain Administrator rights, effectively compromising the
+entire corporate infrastructure.
+
+This rule looks for unusual childs of the `dns.exe` process, which can indicate the exploitation of the SIGRed or a 
+similar remote code execution vulnerability in the DNS server.
+
+#### Possible investigation steps
+
+- Investigate the process execution chain (parent process tree) for unknown processes.
+  - Any suspicious or abnormal child process spawned from dns.exe should be reviewed and investigated with care. It's
+  impossible to predict what an adversary may deploy as the follow-on process after the exploit, but built-in
+  discovery/enumeration utilities should be top of mind (whoami.exe, netstat.exe, systeminfo.exe, tasklist.exe).
+  - Built-in Windows programs that contain capabilities used to download and execute additional payloads should also be
+  considered. This is not an exhaustive list, but ideal candidates to start out would be: mshta.exe, powershell.exe,
+  regsvr32.exe, rundll32.exe, wscript.exe, wmic.exe.
+  - If the DoS exploit is successful and DNS Server service crashes, be mindful of potential child processes related to
+  werfault.exe occurring.
+- Investigate abnormal behaviors observed by the subject process such as network connections, registry or file
+modifications, and any spawned child processes.
+- Investigate other alerts associated with the host during the past 48 hours.
+- Check whether the server is vulnerable to CVE-2020-1350.
```

Comment:
Do you think it'd be helpful to link off to relevant docs for CVE-2020-1350 or docs that explain what CVE-2020-1350 is and how to address it?

Reply:
Added a few actions on the response section